### ISSUE TITLE
Option to omit unnecessary decimal places

### ISSUE THREAD

**Issue by TomOne** [Pephers/Super-Calculator#12]
An option to omit unnecessary decimal places would be great. If for example one has set `"round_decimals": 4`, the result of divisions will always have 4 number of digits after the decimal point, e.g. `10/2 = 5.0000`. I think that is not always desirable, because most people would expect `5` as result in this case, without the 4 zeroes after the decimal point.

I see that in your [usage.gif](https://github.com/Pephers/Super-Calculator/raw/master/images/usage.gif) you get `20.00` as result and then you manually delete the last three characters to get `20`. Wouldn’t it be nicer if that would be handled by the plugin itself? I’m quite sure that hardly anyone would leave `20.00px` in the CSS.

**Comment by alkuzad** [comment]
so just use 10//2

**Comment by wouter-muller** [comment]
I strongly agree with this, especially when you are dealing with CSS, it is a bad practise to have trailing zeros. Also, it looks terrible. 

Would be awesome if the plug-in could be updated to automatically remove trailing zeros!

**Comment by wouter-muller** [comment]
It seems that adding:

```
trim_zeros: true
```

to your Super Calculator user settings does the trick!

**Comment by TomOne** [comment]
Thanks, this works. But it trims every zero, even if it occurs before the decimal point (e.g. `.667` instead of `0.667`). It looks like there’s no option to prevent that.

**Comment by wouter-muller** [comment]
At least in the CSS world, it is considered a best practise to trim leading zeros as well, so `.667` instead of `0.667`.
